test/util.go

```diff
@@ -19,6 +19,14 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+// AssertCanvasTappableAt asserts that the canvas is tappable at the given position.
+func AssertCanvasTappableAt(t *testing.T, c fyne.Canvas, pos fyne.Position) bool {
```

Comment:
Sorry I didn't pick up on this in our conversation but this method does not assert in the same way that other `Assert...` methods do - it's just checking.
I guess that either it should be just `CanvasTappableAt` or it should perform an assertion instead of returning `bool`.

Reply:
Ooops … forgot the assertion!

Reply:
fixed
and added test, too